src/Mono.Android/metadata

```diff
@@ -1953,6 +1953,12 @@
   <attr api-since="35" path="/api/package[@name='android.service.notification']/class[@name='Condition']/constructor[@name='Condition' and count(parameter)=4 and parameter[1][@type='android.net.Uri'] and parameter[2][@type='java.lang.String'] and parameter[3][@type='int'] and parameter[4][@type='int']]/parameter[4]" name="enumType">Android.Service.Notification.ConditionSource</attr>
   <attr api-since="35" path="/api/package[@name='android.service.notification']/class[@name='Condition']/constructor[@name='Condition' and count(parameter)=8 and parameter[1][@type='android.net.Uri'] and parameter[2][@type='java.lang.String'] and parameter[3][@type='java.lang.String'] and parameter[4][@type='java.lang.String'] and parameter[5][@type='int'] and parameter[6][@type='int'] and parameter[7][@type='int'] and parameter[8][@type='int']]/parameter[7]" name="enumType">Android.Service.Notification.ConditionSource</attr>
 
+  <!-- Rename Namespaces -->
+  <attr api-since="36" path="/api/package[@name='android.app.appfunctions']" name="managedName">Android.App.AppFunctions</attr>
+  <attr api-since="36" path="/api/package[@name='android.app.appsearch.functions']" name="managedName">Android.App.AppSearch.Functions</attr>
+  <attr api-since="36" path="/api/package[@name='android.configinfrastructure.aconfig']" name="managedName">Android.ConfigInfrastructure.Aconfig</attr>
```

Comment:
I don't know if we have a past example, but if `aconfig` is short for "Android Config", should the C# namespace be `Android.ConfigInfrastructure.AConfig`?

Reply:
Yeah, I researched this one to see what Google's intent is.  I generally defer to the way they have named their types which in this case is indeed `Aconfig`:

https://developer.android.com/reference/android/configinfrastructure/aconfig/AconfigPackage?hl=en